tests/testit/test-render.R

```diff
@@ -0,0 +1,13 @@
+library(testit)
+
+assert('run_pandoc() find when Pandoc needs to convert', {
+  (run_pandoc(c("nothing special", "requiring render")) %==% FALSE)
+  (run_pandoc(c("```{=html}", "using raw block content", "```")) %==% TRUE)
```

Comment:
I would probably just use `(cond)` or `(!cond)` instead of `(cond %==% TRUE)` or `(cond %==% FALSE)`. It doesn't really matter.

Reply:
I hesitated with it 😅 but I liked the TRUE / FALSE being seen directly without having to decode `(!...)` or `(...)` but it is a matter of habits. I understand: keep things short when possible. I'll apply that.

Reply:
There is an advantage of using `%==%`, though: when the LHS is not exactly TRUE or FALSE, we will see what its value is in the error message. That's why I didn't add a change suggestion.

Reply:
Oh, I had not that in mind. Thanks for the precision.